flake.nix

```diff
@@ -41,7 +41,7 @@
       ];
       perSystem = { config, self', inputs', system, pkgs, lib, ... }:
         let
-          rustToolchain = inputs'.fenix.packages.stable.toolchain;
```

Comment:
No need to drop stable toolchain.

Reply:
I guess you locally used that unstable feature, then applied [this simple fix](https://github.com/rust-lang/rust/issues/62290#issuecomment-1722085100), but forgot to switch back to stable.